src/main/scala/uk/gov/hmrc/ct/accounts/frs10x/boxes/AC8899.scala

```diff
@@ -28,7 +28,9 @@ case class AC8899(value: Option[Boolean]) extends CtBoxIdentifier(name = "Direct
   with DirectorsReportEnabledCalculator {
 
   override def validate(boxRetriever: Frs10xDirectorsBoxRetriever with FilingAttributesBoxValueRetriever): Set[CtValidation] =
-    failIf(boxRetriever.directorsReportEnabled().value) (
-      validateBooleanAsTrue("AC8899", this)
-    )
+    if (directorsReportEnabled(boxRetriever))
```

Comment:
This if with an empty set return on the else is the whole point of failIf
```
failIf(directorsReportEnabled(boxRetriever)) {
    validateBooleanAsMandatory("AC8899", this)
}
```
This should give you the same result.